src/app/login/login.component.css

```diff
@@ -1,94 +1,96 @@
-*{box-sizing:border-box;}
-
 .main {
-    height: 100%;
-    width: 100%;
-    background: #424242;
-    background-image: url("/assets/Kind_auf_Gehweg.jpg");
+  height: 100%;
+  width: 100%;
+  background: hsl(0,0%,26%) url("/assets/Kind_auf_Gehweg.jpg");
 }
 
 .log-form {
   width: 40%;
   min-width: 320px;
   max-width: 475px;
-  background: #fff;
+  background: hsl(0,0%,100%);
   position: absolute;
   top: 50%;
   left: 50%;
-  -webkit-transform: translate(-50%,-50%);
--moz-transform: translate(-50%,-50%);
--o-transform: translate(-50%,-50%);
--ms-transform: translate(-50%,-50%);
-transform: translate(-50%,-50%);
-  
-  box-shadow: 0px 2px 5px rgba(0,0,0,.25);
-  
-  @media(max-width: 40em){
+
+  -webkit-transform: translate(-50%, -50%);
+  -moz-transform: translate(-50%, -50%);
+  -o-transform: translate(-50%, -50%);
+  -ms-transform: translate(-50%, -50%);
+  transform: translate(-50%, -50%);
+
+  box-shadow: 0 2px 5px hsla(0, 0%, 0%, .25);
+}
+
+@media (max-width: 40em) {
+  .log-form {
     width: 95%;
     position: relative;
     margin: 2.5% auto 0 auto;
-    left: 0%;
-  -webkit-transform: translate(0%,0%);
--moz-transform: translate(0%,0%);
--o-transform: translate(0%,0%);
--ms-transform: translate(0%,0%);
-transform: translate(0%,0%);
+    left: 0;
+
+    -webkit-transform: translate(0%, 0%);
+    -moz-transform: translate(0%, 0%);
+    -o-transform: translate(0%, 0%);
+    -ms-transform: translate(0%, 0%);
+    transform: translate(0%, 0%);
   }
 }
```

Comment:
Our CSS-code is pure css and not sass/scss. Nesting is not part of the css spec. Functions, such as `darken` and `lighten` do not exist in css.

I replaced all color codes with `hsl` and `hsla`. The `darken` and `lighten` functions can then be replaced by changes to the lightness parameter. I replaced the `fadeout` function (should be `fade-out`?) with corresponding changes to the alpha values.

Personally, I think sass would be overkill for this project. But If you would prefer to use sass, then we could add it to the build process.